notification2-examples/hello-world-microservice/README.md

```diff
@@ -0,0 +1,149 @@
+# Introduction
+This example microservice demonstrates how to create a subscription and consume a stream of notifications using WebSocket protocol. The example microservice first creates a subscription and then proceeds to connect to the WebSocket server after obtaining the authorization token.
```

Comment:
Maybe this sentence should list the steps in order (mention token creation for the subscription before mentioning connecting)

Reply:
I am going to accept your suggestions below and mark this item as resolved.

Reply:
I can suggest a change for this instead